addons/ingress-nginx/v1.6.0.yaml

```diff
@@ -199,12 +199,7 @@ rules:
   - ""
   resources:
   - events
-  verbs:
-  - create
-- apiGroups:
-  - ""
-  resources:
-  - endpoints
+  - configmaps
```

Comment:
This permissions are too open. Please check https://github.com/kubernetes/ingress/tree/master/examples/rbac/nginx
We only need to update one configmap used to store the leader election.